util/coding.h

```diff
@@ -354,6 +354,13 @@ inline Slice GetSliceUntil(Slice* slice, char delimiter) {
 }
 
 template<class T>
+#ifdef ROCKSDB_UBSAN_RUN
+#if defined(__clang__)
+__attribute__((no_sanitize("alignment")))
```

Comment:
should it be `__no_sanitize__`? I don't have experience with it but out linter is complaining about it.